plugins/module_utils/nd.py

```diff
@@ -516,6 +523,7 @@ def get_diff(self, unwanted=None):
                         del existing[key]
                     except KeyError:
                         pass
+                if key in sent:
                     try:
                         del sent[key]
                     except KeyError:
```

Comment:
Do we still need this try/except on KeyError after adding the key check above?

Reply:
Nope. Changed it..